plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -674,7 +677,12 @@ global proc int mayaUsdTranslatorExport (string $parent,
                         -label "" -label1 $conversion -annotation $ann
                         $widgetName;
                 }
-                
+
+                optionMenuGrp -l `getMayaUsdString("kExportRelativeTexturesLbl")` -annotation `getMayaUsdString("kExportRelativeTexturesAnn")` exportRelativeTexturesPopup;
+                    menuItem -l `getMayaUsdString("kExportRelativeTexturesAutomaticLbl")` -ann "automatic";
+                    menuItem -l `getMayaUsdString("kExportRelativeTexturesAbsoluteLbl")` -ann "absolute";
+                    menuItem -l `getMayaUsdString("kExportRelativeTexturesRelativeLbl")` -ann "relative";

```

Comment:
According to the JIRA item it says, "The tooltip on the dropdown is the same as on the label." With this code they aren't identical (case is different) and once translated the tooltip will only be in English. I think this should be changed to something like:

```
string $lbl;
$lbl = getMayaUsdString("kExportRelativeTexturesAutomaticLbl");
menuItem -l $lbl -ann $lbl;
<etc, for other two>
```

Reply:
Hi @seando-adsk , Can we do this?

Reply:
You can but you have to surround the getMayaUsdString call with ``
But that has extra overhead of calling getMayaUsdString twice for the same string. Why not store it for use twice?

Reply:
Yea I just think this will make the code shorter. I can go with a variable as well

Reply:
Updated UI:
![image](https://github.com/Autodesk/maya-usd/assets/127771016/c29cee86-a0f0-44e7-9f7b-40e910a1666f)

Reply:
The story meant that the tooltip be the same on the label of the drop down ("Texture File Paths") and the drop-down itself. This is similar to other UI elements and other stories. So the tooltip should have been left as it was, with the long text explaining what each choices do.

Reply:
Hi @pierrebai-adsk , yes I have changed that since the annotation is actually used for token string. Check out this PR: https://github.com/Autodesk/maya-usd/pull/3294